protocols/protocol/cloudstate/crdt.proto

```diff
@@ -51,20 +56,41 @@ option go_package = "cloudstate/protocol";
 // Special care must be taken when working with maps and sets. The keys/values are google.protobuf.Any, which encodes
```

Comment:
>and may subsequently be terminated if that entity becomes idle, or if the entity is deleted.

How does an entity become idle?

Reply:
>It may be kept alive and used to handle multiple commands, and may subsequently be terminated if that entity becomes idle, or if the entity is deleted. 

The entity being "terminated" and "a terminated handle stream" do not mean the same, yes?

>Shutdown is typically done for efficiency reasons, unless the entity is explicitly deleted, a terminated handle stream does not mean the proxy has stopped tracking the state of the entity in its memory.

If the handle stream is terminated, what state does the entity have when it's not explicitly deleted? Can a proxy re-establish the handle stream, having an entity still not be deleted? and therefore send a non CrdtInit message over the freshly initiated handle stream (what would run against the obligation to have a CrdtInit always be the first message on a fresh handle stream). 

In the Java/Scala implementation I see a context can go inactive, is that an entity being idle?
Perhaps we could clarify the terms "idle", "terminated entity", "terminated stream", "shutdown" and "deleted" and their implications to the state of an entity to be managed by a user support library.

Reply:
I'm writing some functional tests and I come back to these points, @jroper @pvlugter wdyt here?

See also below the question regarding `CrdtDelete`, if we don't close a stream without a failure or reason, what state would the stream be if the proxy instructed the user function to delete the CRDT?

>A proxy may decide to terminate the stream after sending this.

If the proxy does not decide to terminate the stream, what messages is the user function allowed to receive after a CRDT was deleted? A State message or none?

Reply:
Go Support (v0.2.x) supports the stream to be re-used, clears its state completely and would allow a new init message.